extension/view/js/InPageTranslation.js

```diff
@@ -589,13 +594,29 @@ class InPageTranslation {
 
                         if (!counterpart) {
 
+                            const hasOnlyQEAttributes = node => {
+                                const allowed = new Set([
+                                    "x-bergamot-sentence-index", "x-bergamot-sentence-score",
+                                    "x-bergamot-word-index", "x-bergamot-word-score",
+                                ]);
+
+                                if (node.nodeType === Node.ELEMENT_NODE) {
+                                    if (node.nodeName !== "FONT") return false;
+
+                                    if (!node.getAttributeNames().every(attribute => allowed.has(attribute))) return false;
+
+                                    for (let child of node.childNodes) if (!hasOnlyQEAttributes(child)) return false;
```

Comment:
Iterate over `.children` instead of `.childNodes`.

`.children` contains elements only, `childNodes` also contains non-elements.